listenbrainz/webserver/static/js/src/playlists/Playlist.tsx

```diff
@@ -569,20 +569,17 @@ export default class PlaylistPage extends React.Component<
       return;
     }
     try {
-      // Replace keys that have changed but keep all other attributres
-      const editedPlaylist: JSPFPlaylist = defaultsDeep(
-        {
-          annotation: description,
-          title: name,
-          extension: {
-            [MUSICBRAINZ_JSPF_PLAYLIST_EXTENSION]: {
-              public: isPublic,
-              collaborators,
-            },
+      const editedPlaylist: JSPFPlaylist = {
```

Comment:
This code I copied form the [Playlists.tsx component](https://github.com/metabrainz/listenbrainz-server/blob/5b5ad5d11500ee8b4f64caf65b3a5c3f8fd67624/listenbrainz/webserver/static/js/src/playlists/Playlists.tsx#L339-L349), which is working contrarily to the `defaultsDeep` replacement which turned out to be a bad idea.